assets/src/scss/layout/_navbar.scss

```diff
@@ -452,20 +326,6 @@ a.nav-link {
   }
 }
 
-.top-navigation-overlay {
```

Comment:
It's been moved to the burger menu styles. It's not necessary to add it here.

Reply:
I no longer see the overlay when opening the side menu on tablet, but from this comment I'm not sure if that's because it's defined in another PR?